VisualStudio.gitignore

```diff
@@ -39,9 +39,10 @@ Generated\ Files/
 [Tt]est[Rr]esult*/
 [Bb]uild[Ll]og.*
 
-# NUNIT
+# NUnit
 *.VisualState.xml
 TestResult.xml
+nunit-*.xml
```

Comment:
Any links to StackOverflow discussions or other documentation about this change?

Reply:
Here they are:
- https://github.com/nunit/docs/wiki/XML-Formats
- https://github.com/github/VisualStudio/blob/master/.gitignore